### DISCUSSION TITLE
Help getting started with Pi

### DISCUSSION THREAD

**Discussion by Kelaita50** [nasa-jpl/open-source-rover#488] [General]
Hello I need help getting the code and working with it. Can someone be willing to Zoom or Teams to assist me even if it is 10 minutes

**Comment by Achllle** [comment]
Hi! Please join the slack community by clicking on the 'Slack' image in the main [README](https://github.com/nasa-jpl/open-source-rover?tab=readme-ov-file#join-a-community-of-hundreds-of-builders). Then post your question, what you've tried, and what errors you run into on the #troubleshooting channel. This will get you help the fastest, especially since there are over 500 people there. If it turns out that we need to do a screen share, we'll do that then but you have to first try via Slack.
